src/components/recipes/JSONArgumentsField.js

```diff
@@ -0,0 +1,35 @@
+import { Input } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import FormItem from 'console/components/forms/FormItem';
+
+export default class JSONArgumentsField extends React.Component {
+  static propTypes = {
+    disabled: PropTypes.bool,
+    recipeArguments: PropTypes.instanceOf(Map),
+  };
+
+  static defaultProps = {
+    disabled: false,
+    recipeArguments: new Map(),
+  };
+
+  render() {
+    const { disabled, recipeArguments } = this.props;
+
+    return (
+      <div>
+        <p className="action-info">Contextual UI is not available for this action.</p>
```

Comment:
This, I'm not sure it helps. It makes it look broken. Perhaps I'm not just understanding what it means.

Reply:
I'll expand on it so it's a little clearer:
```
Contextual UI is not available for this action. We have provided a generic editor instead.
```